src/rshim_fuse.c

```diff
@@ -1305,6 +1328,10 @@ int rshim_fuse_init(rshim_backend_t *bd)
      * device was re-ceated during SW_RESET.
      */
     snprintf(buf, sizeof(buf), "/dev/rshim%d/%s", bd->index, name);
+    if (i == RSH_DEV_TYPE_MISC && !access(buf, F_OK)) {
+      /* It's OK for misc file to be persistent after fuse_del() */
+      continue;
+    }
```

Comment:
Any reason we need this logic? It might cause issue for next restart. The fuse file will cause trouble if no rshim driver supports it.

Reply:
After the driver releases rshim upon other side's request, it still needs to keep the "misc" device node so the user can cat "FORCE_CMD" to it. I agree this needs to be thoroughly tested.